packages/graphql-server/src/resolvers/appointment/resolvers.ts

```diff
@@ -0,0 +1,52 @@
+import appointmentServices from './services'
+import { checkPermission } from '../../utils/check-permission'
+import logger from '../../logger'
+import errors from '../../errors'
+import { ServerContext } from '../../app'
+import {
+  GetAppointmentByIdArgs,
+  GetAppointmentsArgs,
+  UpdateAppointmentArgs,
+  CreateAppointmentArgs,
+} from './appointments'
+
+export const queryAppointment = (_: any, args: GetAppointmentByIdArgs, context: ServerContext) => {
```

Comment:
Please add return types to these functions

Reply:
Because the function `callGetAppointmentByIdAPI` contains appropriate type definition, this function's type will be interpreted automatically

Reply:
See my previous, please be explicit with your type defintions